src/schema.js

```diff
@@ -23,9 +23,15 @@ let fromEsIndexMapping = _.mapValues(
     _.get('mappings'),
     // Always 1 type per index but sometimes there's a `_default_` type thing
     _.omit(['_default_']),
-    _.toPairs,
-    // Capture esType
-    ([[type, fields]]) => ({ fields, elasticsearch: { type } }),
+    x => _.size(_.keys(x)) == 1
```

Comment:
This is the elasticsearch equivalent of duck typing 😅 

We may want to just drop ES 6 support since this is a breaking change, but this is one of only two things needed to maintain ES 6 support.